docs/known-issues/aspire-local-packages.md

```
@@ -0,0 +1,75 @@
+System.AggregateException: One or more errors occurred. 
+  (Property CliPath: The path to the DCP executable used for Aspire orchestration is required.; 
+   Property DashboardPath: The path to the Aspire Dashboard binaries is missing.)
+```
+
+## Causa Raiz
+
+Este é um bug conhecido no .NET Aspire 13.x quando os pacotes NuGet são armazenados em um diretório customizado (usando `globalPackagesFolder` no nuget.config).
+
+- O MSBuild corretamente define as propriedades `AspireDashboardPath` e `DcpCliPath`
+- Mas o código runtime do Aspire espera `DashboardPath` e `CliPath` (sem prefixo "Aspire"/"Dcp")
+- Issue rastreada em: https://github.com/dotnet/aspire/issues/6789
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)